src/mixer.erl

```diff
@@ -105,6 +107,16 @@ expand_mixin(Line, Name) when is_atom(Name) ->
             [#mixin{line=Line, mod=Name, fname=Fun, alias=Fun, arity=Arity} || {Fun, Arity} <- Exports,
                                                                                Fun /= module_info]
     end;
+expand_mixin(Line, {Name, except, Funs}) when is_atom(Name),
+                                              is_list(Funs) ->
+    case catch Name:module_info(exports) of
```

Comment:
Remove the dynamic function call on line 112. Only modules that define callbacks should make dynamic calls.